book.toml

```diff
@@ -7,3 +7,9 @@ title = "PermaplanT"
 
 [preprocessor.generate-summary]
 get_chapter_name_from_file = true
+
+[preprocessor.mermaid]
+command = "mdbook-mermaid"
+
+[output.html]
+additional-js = ["mermaid.min.js", "mermaid-init.js"]
```

Comment:
Please put these files in a sub-folder of doc. We want to keep top-level clean.

Reply:
I will put them directly into doc. I cannot create a subfolder in doc, otherwise the pre-processor would include that folder in the book.